tests/e2e/specs/customizer/breadcrumbs/after-enable-breadcrumb-on-various-pages.test.js

```diff
@@ -0,0 +1,117 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/customize';
+describe( 'Breadcrumb settings in the customizer', () => {
+	it( 'enable breadcrumb on home page for after header position should apply corectly for inside position', async () => {
+		const afterBreadcrumb = {
+			'breadcrumb-position': 'astra_header_after',
+			'breadcrumb-disable-home-page': 0,
+		};
+		await setCustomize( afterBreadcrumb );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-breadcrumbs-wrapper' );
+		const enablehomePage = await page.$eval( '.ast-breadcrumbs-wrapper', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```

Comment:
const declare as camelCase for enablehomePage
```suggestion
		const enableHomePage = await page.$eval( '.ast-breadcrumbs-wrapper', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```

Reply:
Done changes